tests/VCR/Storage/YamlTest.php

```diff
@@ -62,7 +62,7 @@ public function testIterateFirstNestedObject()
     public function testIterateSecondNestedObject()
     {
         $this->iterateAndTest(
-            "-". "\n"
+            "-" . "\n"
```

Comment:
canso you also check if there is any change if you enable `braces`? I feel like some old if statements without braces are not being fixed, might be an impression tho!